MotionMark/resources/debug-runner/debug-runner.js

```diff
@@ -22,34 +22,35 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
-ProgressBar = Utilities.createClass(
-    function(element, ranges)
+
+class ProgressBar {
```

Comment:
I think this class should be removed. I recall showing it in the page affects the score. We may need to show the sub-test name and its order in the running suites, e.g. `Multiply (1/8)`, `Canvas Arcs (2/8)`, etc.

Reply:
Right; it's unused at present I think.